cypress/fixtures/adFormat.json

```diff
@@ -0,0 +1,11 @@
+{
+	"ad_unit_formats":[
+		{
+			"id":11,
+		"approved_for_direct":1,
+		"approved_for_pmp":1
+			
+		}
+		]
+	
+}
```

Comment:
* This does not look like a properly formatted json, pls fix
* Add space at the end